projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -54,6 +54,38 @@ public function init_actions() {
 		add_action( 'switch_theme', array( $this, 'delete_cache' ) );
 		add_action( 'wp_trash_post', array( $this, 'delete_on_post_trash' ), 10, 2 );
 		add_filter( 'wp_php_error_message', array( $this, 'disable_caching_on_error' ) );
+
+		/**
+		 * Delete all cache.
+		 *
+		 * Allow third-party plugins to clear all cache.
+		 */
+		add_action( 'clear_page_cache_all', array( $this, 'delete_cache' ) );
```

Comment:
We need to prefix the actions. I guess my dreams of having all cache plugins using the same actions will have to wait for another day...

```suggestion
		add_action( 'jetpack_boost_clear_page_cache_all', array( $this, 'delete_cache' ) );
```